bin/requirements.yml

```diff
@@ -19,3 +19,6 @@
 
 - src: geerlingguy.postgresql
   version: 1.4.5
+
+- src: libre_ops.wal2json
```

Comment:
Who is exactly does not understand WAL and needs JSON instead?

Reply:
Everybody needs JSON! :trollface: 

Basically:
```
Postgres -> Wal2json ->  Kafka queue -> JDBC driver -> Postgres replica
```

But it also means that other things can read those database changes from the queue and parse them easily.